### PR TITLE
RequestDto changed to responseDto

### PR DESCRIPTION
**Pull request by romaanankin**
Used source: https://developers.google.com/youtube/v3/docs/videos/list

### CODE REVIEW

File: src/main/java/mate/academy/wikibot/dto/ResponseDto.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package mate.academy.wikibot.dto;
+
+public class ResponseDto {
+    private String kind;
+    private String videoId;
+    private String title;
+    private String url;
+}
+
```
Comment (BondarenkoJek) [inline src/main/java/mate/academy/wikibot/dto/ResponseDto.java]: delete lines

File: src/main/java/mate/academy/wikibot/dto/ResponseDto.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package mate.academy.wikibot.dto;
+
+public class ResponseDto {
```
Comment (BondarenkoJek) [inline src/main/java/mate/academy/wikibot/dto/ResponseDto.java]: where getters/setters and etc?
or add @Data

Comment (romaanankin) [reply src/main/java/mate/academy/wikibot/dto/ResponseDto.java]: where are=)

**Comment by romaanankin** [comment]
> Why did you deleted requestDto? As i nderstand yhe task you should change one for another...

**Pull request closed without merging**